src/osd/PrimaryLogPG.cc

```diff
@@ -13331,6 +13430,44 @@ void PrimaryLogPG::hit_set_persist()
   simple_opc_submit(std::move(ctx));
 }
 
+HitSetRef PrimaryLogPG::hit_set_load(hobject_t& oid) {
```

Comment:
Is the reason hit_set_load() is needed here but not for the other types because for the temp type there is always just a single hit set that is reused (vs a history of them)?

Reply:
Yes, all historical information is recorded in a single temp hit set. So we can always reuse it.